misc/test_gpu_sm_op.py

```diff
@@ -22,29 +22,26 @@
 cols = N - 1
```

Comment:
Shall we simply remove this file as it is added to tests?

Reply:
Sure!